### PR TITLE
Adding Ownership Rule "module" documentation

### PR DESCRIPTION
**Pull request by iProgramStuff**

### CODE REVIEW

File: src/docs/product/error-monitoring/issue-owners/index.mdx [hunk 1]
```diff
@@ -23,14 +23,15 @@ You define ownership rules per project. To configure ownership rules, navigate t
...
+2. Module: matches against the module in the event's stack trace
+3. URL: matches against the event's `url` tag
+4. Tag: matches against event tags
 
 The general format of a rule is: `type:pattern owners`
 
 `type`
 
-: Can be either `path`, which matches on paths, `url`, which matches the specified URL, or `tags.TAG_NAME` which matches the specified tag.
+: Can be either `path`, which matches on paths, `module`, which matches on module, `url`, which matches the specified URL, or `tags.TAG_NAME` which matches the specified tag.
```
Comment (PeloWriter) [inline src/docs/product/error-monitoring/issue-owners/index.mdx:34]: These explanations weren't terribly helpful to start. I'd suggest we either enhance them (better explain what path, module, and URL mean in this context) or remove them altogether.

Comment (iProgramStuff) [reply src/docs/product/error-monitoring/issue-owners/index.mdx:34]: Lol I was thinking similarly. The description really doesn't add much above the keyword.

Comment (PeloWriter) [reply src/docs/product/error-monitoring/issue-owners/index.mdx:34]: @iProgramStuff - shall we simply remove these descriptions? They seem pretty self explanatory except for module.

Comment (iProgramStuff) [reply src/docs/product/error-monitoring/issue-owners/index.mdx]: Yes, I think we should. What do you think of the latest copy? :)

File: src/docs/product/error-monitoring/issue-owners/index.mdx [hunk 2]
```diff
@@ -23,14 +23,15 @@ You define ownership rules per project. To configure ownership rules, navigate t
...
+2. Module: matches against the module in the event's stack trace
+3. URL: matches against the event's `url` tag
+4. Tag: matches against event tags
 
 The general format of a rule is: `type:pattern owners`
 
 `type`
 
-: Can be either `path`, which matches on paths, `url`, which matches the specified URL, or `tags.TAG_NAME` which matches the specified tag.
+: Can be either `path`, `module`, or `url`, or `tags.TAG_NAME` which matches the specified tag.
```
Comment (imatwawana) [inline src/docs/product/error-monitoring/issue-owners/index.mdx:34]:
```suggestion
: Can be either `path`, `module`, `url`, or `tags.TAG_NAME`, which matches the specified tag.
```

File: src/docs/product/error-monitoring/issue-owners/index.mdx [hunk 3]
```diff
@@ -23,14 +23,15 @@ You define ownership rules per project. To configure ownership rules, navigate t
 Types of matchers available:
 
 1. Path: matches against all file paths in the event's stack trace
-2. URL: matches against the event's `url` tag
-3. Tag: matches against event tags
+2. Module: matches against the module in the event's stack trace
+3. URL: matches against the event's `url` tag
```
Comment (imatwawana) [inline src/docs/product/error-monitoring/issue-owners/index.mdx:27]:
```suggestion
3. URL: matches against the event's `url`
```

**Review by imatwawana — APPROVED** [review]
LGTM - minor edits to avoid confusion

**Pull request merged**